public/app/core/components/AppChrome/TopSearchBar.tsx

```diff
@@ -56,17 +27,11 @@ export function TopSearchBar() {
         </a>
       </div>
       <div className={styles.searchWrapper}>
-        <FilterInput
-          onClick={onOpenSearch}
-          placeholder="Search Grafana"
-          value={query.query ?? ''}
-          onChange={onSearchChange}
-          className={styles.searchInput}
-        />
+        <TopSearchBarInput />
       </div>
       <div className={styles.actions}>
         {helpNode && (
-          <Dropdown overlay={<TopNavBarMenu node={helpNode} />}>
+          <Dropdown overlay={() => <TopNavBarMenu node={helpNode} />}>
```

Comment:
Hi @torkelo, I would like to know the idea why this is changed to anonymous function? thanks

Reply:
@lpskdl this is what makes the menu "lazy" rendered only when it is opened.